docs/admin/reference/kubernetes/config-secret-keys.md

```diff
@@ -3,7 +3,138 @@ tags:
   - administration
   - reference
   - kubernetes
+  - configmap
+  - secret
 ---
 
-This is a placeholder
+## Overview
+
+The Config Init job creates a set of secrets and configurations used by all Jans services.
+
+To check the values of the configmaps on the current deployment run:
+
+```bash
+kubectl get configmap -n jans -o yaml
+```
+
+To check the values of the secrets on the current deployment run :
+
+```bash
+kubectl get secret -n jans -o yaml
+```
+
+## Jans Configmaps
+
+| Key                                           | Example Values                                     |
+| --------------------------------------------- | -------------------------------------------------- |
+| `admin_email`                                 | `support@jans.io`                                  |
+| `admin_inum`                                  | `631e2b84-1d3d-4f28-9a9a-026a25febf44`             |
+| `auth_enc_keys`                               | `RSA1_5 RSA-OAEP`                                  |
+| `auth_key_rotated_at`                         | `1669143906`                                       |
+| `auth_legacyIdTokenClaims`                    | `false`                                            |
+| `auth_openid_jks_fn`                          | `/etc/certs/auth-keys.jks`                         |
+| `auth_openid_jwks_fn`                         | `/etc/certs/auth-keys.json`                        |
+| `auth_openidScopeBackwardCompatibility`       | `false`                                            |
+| `auth_sig_keys`                               | `RS256 RS384 RS512 ES256 ES384 ES512 PS256 PS384 PS512`|
+| `city`                                        | `Austin`                                           |
+| `country_code`                                | `US`                                               |
+| `CN_AUTH_APP_LOGGERS`                         | `{"audit_log_level":"INFO","audit_log_target":"FILE","auth_log_level":"INFO","auth_log_target":"STDOUT","http_log_level":"INFO","http_log_target":"FILE","ldap_stats_log_level":"INFO","ldap_stats_log_target":"FILE","persistence_duration_log_level":"INFO","persistence_duration_log_target":"FILE","persistence_log_level":"INFO","persistence_log_target":"FILE","script_log_level":"INFO","script_log_target":"FILE"}` | 
```

Comment:
We aren't looking for including the environmental vars for the configmap that are added from the helm chart. We are looking for the entries coming out from the configurator job ( container) which generates a secret and configmap outside the control of helm called normal `cn`. You must install all jans services so you get all the possible entries in both the configmap and secret